### PR TITLE
Introduction of the webots serial driver for RU communication

### PR DESCRIPTION
**Pull request by BlueAndi**

### CODE REVIEW

File: lib/APPTest/App.cpp
Comment (gabryelreyes) [inline lib/APPTest/App.cpp:1]: Please check the file structure of the other applications and repeat it here. Source and header files should be inside a src folder

File: platformio.ini [hunk 1]
```diff
@@ -72,15 +72,26 @@ build_flags =
...
     HALInterfaces
     ArduinoNative
     HALSim
+    Webots
 lib_ignore =
     HALTarget
     HALTest
 extra_scripts =
+    ./scripts/webots_launcher.py
     pre:./scripts/add_os_specific_build_flags.py
```
Comment (gabryelreyes) [inline platformio.ini:89]: Script `scripts\add_os_specific_build_flags.py` can now be removed, as it is only necessary for socket connection

File: scripts/add_os_specific_build_flags.py
Comment (gabryelreyes) [inline scripts/add_os_specific_build_flags.py:1]: No longer necessary

**Review by gabryelreyes — CHANGES_REQUESTED** [review]

File: lib/APPTest/App.cpp
Comment (BlueAndi) [reply lib/APPTest/App.cpp:1]: ok

File: platformio.ini (hunk 1 above)
Comment (BlueAndi) [reply platformio.ini:89]: ok

File: scripts/add_os_specific_build_flags.py
Comment (BlueAndi) [reply scripts/add_os_specific_build_flags.py:1]: ok

**Review by gabryelreyes — APPROVED** [review]

**Pull request merged**